### PR TITLE
remove  ngsiv1 from updateAction

### PR DESCRIPTION
**Pull request by AlvaroVega**
related with issue https://github.com/telefonicaid/perseo-fe/issues/714

Do not merge until ~https://github.com/telefonicasc/iotp-pqa/pull/67~ https://github.com/telefonicasc/iotp-pqa/pull/78

### CODE REVIEW

**Comment by fgalan** [comment]
> Do not merge until ~[telefonicasc/iotp-pqa#67](https://github.com/telefonicasc/iotp-pqa/pull/67)~ [telefonicasc/iotp-pqa#78](https://github.com/telefonicasc/iotp-pqa/pull/78)

That PR has been merged.

File: test/component/metrics/metrics_actions_test.js [hunk 1]
```diff
@@ -231,8 +232,8 @@ describe('Metrics', function() {
                                 should.equal(m.services.unknownt.sum.actionEntityUpdate, 1);
                                 should.equal(m.services.unknownt.sum.okActionEntityUpdate, 0);
                                 should.equal(m.services.unknownt.sum.failedActionEntityUpdate, 1);
-                                should.equal(m.services.unknownt.sum.outgoingTransactions, 1);
-                                should.equal(m.services.unknownt.sum.outgoingTransactionsErrors, 1);
+                                //should.equal(m.services.unknownt.sum.outgoingTransactions, 1);
+                                //should.equal(m.services.unknownt.sum.outgoingTransactionsErrors, 1);
```
Comment (fgalan) [inline test/component/metrics/metrics_actions_test.js:236]: Lefover comments? Remove/recover?

File: CHANGES_NEXT_RELEASE [hunk 2]
```diff
@@ -1 +1 @@
-
+- Remove ngsiv1 support for updateAction (#714)
```
Comment (fgalan) [inline CHANGES_NEXT_RELEASE:1]: The following should be removed from https://github.com/telefonicaid/perseo-fe/blob/task/remove_update_ngsiv1/docs/API/plain_rules.md#update-attribute-action

> version: optional, The NGSI version for the update action. Set this attribute to 2 or "2" if you want to use NGSv2 format. 2 by default. It is NOT recommended to use 1.

Comment (AlvaroVega) [reply CHANGES_NEXT_RELEASE:1]: Done in https://github.com/telefonicaid/perseo-fe/pull/717/commits/0c53acfe53164d9651b82eedf06790cf32599645

File: test/component/metrics/metrics_actions_test.js (hunk 1 above)
Comment (AlvaroVega) [reply test/component/metrics/metrics_actions_test.js]: Fixed in https://github.com/telefonicaid/perseo-fe/pull/717/commits/205e7238267077fe66931465fa0ee4f1e0e5d23c

**Review by fgalan — APPROVED** [review]
LGTM

**Pull request merged**